jumpscale/sals/reservation_chatflow/reservation_chatflow.py

```diff
@@ -418,8 +418,8 @@ def get_solutions_explorer(self, deployed=True):
                 if "form_info" not in metadata:
                     solution_type = self.check_solution_type(reservation)
                 else:
-                    solution_type = metadata["form_info"]["chatflow"]
-                    metadata["form_info"].pop("chatflow")
+                    solution_type = metadata["form_info"].get("chatflow", SolutionType.Unknown.value)
+                    metadata["form_info"].pop("chatflow", None)
```

Comment:
why not do this in one line?
                   ` solution_type = metadata["form_info"].pop("chatflow", SolutionType.Unknown.value)` 

?

Reply:
Thanks. updated